recipes/fenicsx-beat/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+python_min:
```

Comment:
Instead of this file you can add `{% set python_min = "3.10" %}` to the top of the recipe. It is a matter of taste, you don't need to change it. I do prefer it b/c it keeps everything in a single file and reminds me to update it when updating the recipe.

Reply:
I totally agree. Fixed in https://github.com/conda-forge/staged-recipes/pull/29500/commits/677a7b8b0907f1e1b175f1a426974263db2e86e4